lib/client.js

```diff
@@ -132,6 +135,14 @@ Client.prototype.getJwtFromUrl = function () {
 };
 
 /**
+ * Attempts to replace the JWT in the url with the new JWT passed in the function.
+ * @param {string} newJwt
+ */
+Client.prototype.replaceJWTInUrl = function (newJwt) {
+  window.location.hash = '/?jwt=' + newJwt;
```

Comment:
The original JWT is extracted from `window.location.href`, which doesn't necessarily have to be the hash, even though that is what ID Site is using. However, updating other parts of the URL other than the hash will cause it to reload and then the new JWT will be consumed again, so I'm not sure how to solve this problem.